content/en/monitors/guide/on_missing_data.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Migrating to On Missing Data Configuration
+further_reading:
+- link: "/api/latest/monitors/"
+  tag: "API"
+  text: "Monitors API Documentation"
+---
+
+## Overview
+
+Metric Monitors offer enhanced options for handling missing data, allowing you to differentiate between missing data as a failure mode and a healthy state. 
+
+These options align with what's available in other monitors types like CI, Database, Error Tracking, and more. They apply to query-based monitors, including Integration and Anomaly monitors, providing two main configurations based on the aggregation type:
+
+## Benefits of using On Missing Data options
+
+Metric Monitors tracking errors should reflect as "OK" when no errors occur. With the legacy No Data configurations, monitors would raise alerts with missing data, leading to confusion and workarounds. The On Missing Data configuration options allow monitors to reflect health states, eliminating the need for workarounds.
+
+## Monitors managed through the UI
+
+If you manage your monitors from the UI, they will be automatically updated for you. If you'd like to update them sooner, you can do it through the API as explained below.
+
+## Monitors managed through the API or Terraform
+
+If you are managing your monitors with API or Terraform, replace `notify_no_data` and `no_data_timeframe` with `on_missing_data`. The `no_data_timeframe` parameter is not required since `on_missing_data` uses the same timeframe as the time window.  
```

Comment:
```suggestion
If you are managing your monitors with API or Terraform, replace `notify_no_data` and `no_data_timeframe` with `on_missing_data`. The `no_data_timeframe` parameter is not required since `on_missing_data` uses the same time frame as the time window.  
```